_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_par.cpp

```diff
@@ -2908,6 +2908,23 @@ void SetDefaults(
     if (IsOff(CO3.EnableQPOffset))
         Zero(CO3.QPOffset);
 
+    if (CO2.MinQPI || CO2.MinQPP || CO2.MinQPB ||
```

Comment:
why do we need this condition?

Reply:
Don't refer internal tools.
Most likely "if any is initialized, other should be initialized too".